reactioncommerce/reaction
# refactor: move `template` from server/no-meteor to node-app

Resolves #5627 
Impact: **minor**  
Type: **refactor**

As part of our effort to separate our Reaction Admin from our Reaction Node API, we are moving our GraphQL services from their respective `/imports/plugins/templates/server/no-meteor/` folders into their new `/node-app/plugins/templates` folder.

Updates also include future-proofing some existing code, including adding filetype extensions (and `index.js` to imports)

## Breaking changes
Any custom plugins that import files directly from the `templates` package will need to be updated.

## Testing
1. Start the application and see that it works as it should.
1. See that the `Templates` collection is registered in `context.collections`
